### ISSUE TITLE
ngLocale i18n files FIRSTDAYOFWEEK and WEEKENDRANGE

### ISSUE THREAD

**Issue by poshest** [angular/angular.js#11900]
Are some of the values for FIRSTDAYOFWEEK and WEEKENDRANGE in the [ngLocale files](https://github.com/angular/angular.js/tree/master/src/ngLocale) wrong? Or do I just not understand them?  In which case, this is a call for documentation!  Full question on [Stackoverflow](http://stackoverflow.com/questions/30076505/angularjs-i18n-nglocale-firstdayofweek-and-weekendrange)

**Comment by lgalfaso** [comment]
Monday is day 0 as specified by ISO-8601

**Comment by poshest** [comment]
Oh, really?  Well, that would explain it then.  Hmm, in Javascript, getDay() returns 0 = Sunday...  Why did you choose a different standard?

In any case thanks! It would be great if the docs mentioned this somewhere. I couldn't find "ISO" mentioned on any of the 3 first Google results for "ngLocale".

**Comment by lgalfaso** [comment]
ISO-8601 is not perfect, and even when it differs with Javascript day numbering it is widely used

**Comment by poshest** [comment]
On further reflection, I really do think that the Javascript scheme (0=Sunday, ...) should be used.  The first thing everyone will do is 

    if ($locale.DATETIME_FORMATS.FIRSTDAYOFWEEK === myDate.getDay()) {...}

**Comment by lgalfaso** [comment]
@poshest I think we will have to disagree here. The numbering goes all the way back to the CLDR, so unless we move to a newer version of the standard that uses named days and not numbers, then I would stick to the current definition

**Comment by tiaoshi** [comment]
I came across some locales that I think may be wrong. 

In the locale file for PR China (zh-cn), the first day of the week is 6, so it is Sunday, right? But according to a published [standard](http://www.pep.com.cn/xiaoyu/jiaoshi/study/jszy/bzgfbh/szyfbzgf/201012/P020101203612911209242.pdf) (in Chinese), the first day should be Monday. 

I also took a look at the locale for GB (en-gb), and the first day of the week is Monday (0). I am not sure but I think it should be Sunday (6) for Britain. Could anyone help clarify this?
